pkg/infinispan/grafana/constants.go

```diff
@@ -0,0 +1,7 @@
+package grafana
+
+// Constants for a community Infinispan Grafana installation
+const (
+	ApplicationName = "infinispan"
+	MonitoringKey   = "middleware"
+)
```

Comment:
Is this required in a separate `.go` file so that it can be easily replaced downstream?

Reply:
Values that need downstream changes should be stored in the [csv](https://github.com/infinispan/infinispan-operator/blob/3c4554b970ae9b179f811e72c8e81028e51dcc33/deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml#L134) file and get from the env (i.e. [here](https://github.com/infinispan/infinispan-operator/blob/3c4554b970ae9b179f811e72c8e81028e51dcc33/pkg/controller/constants/image.go#L26)).
All downstream customizations are done there atm.

Reply:
Alright, I will look at that. I just copied what keycloak had done.